app/views/contacts/index.html.erb

```diff
@@ -4,18 +4,39 @@
 
   <p><%= button_to 'View my requests', user_contact_requests_path(current_user), method: :get %></p>
 
-  <table>
-    <thead>
-    <tr>
-      <th>Email</th>
-    </tr>
-    </thead>
-    <tbody>
-    <% @contacts.each do |contact| %>
-      <tr>
-        <td><%= contact.email %></td>
-      </tr>
-    <% end %>
-    </tbody>
-  </table>
+  <% @contacts.each do |contact| %>
+   <div class="card">
+     <div class="card-body">
+       <div class="container">
+         <div class="row">
+           <div class="col">
+             <img src="//lh3.googleusercontent.com/-6V8xOA6M7BA/AAAAAAAAAAI/AAAAAAAAAAA/rzlHcD0KYwo/photo.jpg?sz=120" class="rounded-circle" style="width: 50px">
+           </div>
+           <div class="col-10">
+             <h5 class="card-title">Herrmann<%= contact.firstname %></h5>
```

Comment:
Why do all contacts are named 'Herrmann'. IIRC the contact feature is already implemented and this view can show real values.

Reply:
Oh, it's just because the demo user doesn't have a name, so I wanted to show one for demo purposes. But this gets deleted